docs/content/design/coordinator.md

```diff
@@ -188,6 +188,15 @@ Returns all rules for a specified datasource and includes default datasource.
 
  Returns audit history of rules. default value of interval can be specified by setting `druid.audit.manager.auditHistoryMillis` (1 week if not configured) in coordinator runtime.properties
 
+#### Namespaced Lookups
+
+* `/druid/coordinator/v1/lookups`
```

Comment:
Code is wrong. Will fix

On Thu, Jun 18, 2015, 8:17 AM Slim <notifications@github.com> wrote:

> In docs/content/design/coordinator.md
> <https://github.com/druid-io/druid/pull/1259#discussion_r32741278>:
>
> > @@ -188,6 +188,15 @@ Returns all rules for a specified datasource and includes default datasource.
> >
> >   Returns audit history of rules. default value of interval can be specified by setting `druid.audit.manager.auditHistoryMillis` (1 week if not configured) in coordinator runtime.properties
> >
> > +#### Namespaced Lookups
> > +
> > +* `/druid/coordinator/v1/lookups`
>
> is this suppose to be /druid/coordinator/v1/namespaces as in code
> <https://github.com/metamx/druid/blob/queryTimeLookup/server/src/main/java/io/druid/server/http/NamespacesResource.java#L45-45>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/druid-io/druid/pull/1259/files#r32741278>.
>